torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8867,7 +8867,7 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
                     )),
     BinaryUfuncInfo('mul',
                     aliases=('multiply',),
-                    dtypes=all_types_and_complex_and(torch.float16, torch.bfloat16, torch.bool),
+                    dtypes=all_types_and_complex_and(torch.chalf, torch.float16, torch.bfloat16, torch.bool),
```

Comment:
Nice -- note that test_default_dtypes is now gone in case you want to update some other chalf tests